src/JsonModelBinder/JsonModelBinder.cs

```diff
@@ -42,6 +42,12 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             // Get the json serialized value as string
             string serialized = valueProviderResult.FirstValue;
 
+            // Return a successful binding for empty strings or nulls
+            if (String.IsNullOrEmpty(serialized)) {
+                bindingContext.Result = ModelBindingResult.Success(null);
```

Comment:
Wouldn't it be better to return `ModelBindingResult.Fail()` in case of a null value? @Drenalol

Reply:
But if we return Fail() we dont invoke Controller and return only 400 Error with ModelState.